### ISSUE TITLE
All data should be preserved by saving collections in .json files.[1.5XPts]

### ISSUE THREAD

**Issue by Cisco-the-wayword** [jlvFlores/catalog-of-my-things#23]
All the data pre-user-inputs should be stored in appropriate, well-labelled .json files for easy accessibility
